File: check_db.js
const { Client } = require('pg');

const client = new Client({
  connectionString: 'postgresql://neondb_owner:npg_er9wkZV8IzyE@ep-restless-voice-albuug7e-pooler.c-3.eu-central-1.aws.neon.tech/neondb?sslmode=require'
});

async function run() {
  try {
    await client.connect();
    
    console.log("--- Checking Database Migration Status ---");
    
    const habitsRes = await client.query('SELECT count(*) FROM habits');
    console.log(`Total Habits in DB: ${habitsRes.rows[0].count}`);
    
    const usersRes = await client.query('SELECT id, email, google_user_id FROM users');
    console.log(`Total Users in DB: ${usersRes.rows.length}`);
    
    console.log("\n--- Users Details ---");
    usersRes.rows.forEach(u => {
      console.log(`Email: ${u.email} | Google ID: ${u.google_user_id} | Internal ID: ${u.id}`);
    });
    
  } catch (err) {
    console.error('Error querying database:', err);
  } finally {
    await client.end();
  }
}

run();

File: check_db.py
import sys
import os
sys.path.append(os.path.join(os.path.dirname(__file__), 'backend'))
from app.core.database import SessionLocal
from app.models.habit import Habit
from app.models.user import User

def check_db():
    db = SessionLocal()
    try:
        users = db.query(User).all()
        habits = db.query(Habit).all()
        print(f"Total Users: {len(users)}")
        print(f"Total Habits: {len(habits)}")
        
        print("\n--- Users ---")
        for u in users:
            print(f"ID: {u.id}, Email: {u.email}, Google ID: {u.google_user_id}")
            
        print("\n--- Habits (First 5) ---")
        for h in habits[:5]:
            print(f"ID: {h.id}, Name: {h.name}, User ID: {h.user_id}")
            
    finally:
        db.close()

if __name__ == "__main__":
    check_db()
